### PR TITLE
[32] Update Python version and use Poetry GA

### PR DESCRIPTION
**Pull request by samurang87**
### Issue

#32

### Description

With this PR:
- The CI runs with Python 3.7 and 3.8
- Poetry is installed with a Github Action
- Issue and PR templates are added

### Testing

Will test running the CI 

### Checklist

~- [ ] Documentation (README, docstrings) has been updated~
- [x] Changes in the code are reflected in the CI configuration

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<!--- Describe your changes in detail -->
+
+
+### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+
+
+### Testing
+<!--- Please describe in detail how you tested your changes (e.g. manual changes to the cloud function in the GCP GUI). -->
+<!--- Add screenshots if appropriate, or a link to your testing Grafana dashboard -->
```
Comment (MisterRios) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Grafana Dashboard in logicipi?
```suggestion
<!--- Add screenshots if appropriate -->
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+**Current behaviour**
+A clear and concise description of what the bug is.
+
+**Expected behaviour**
+A clear and concise description of what you expected to happen.
+
+**Where is it observable?**
+Add links if applicable
```
Comment (MisterRios) [inline .github/ISSUE_TEMPLATE/bug.md:17]:
```suggestion
Add links if applicable.
```

**Review by MisterRios — APPROVED** [review]
The PR title does not reflect the addition of the templates, btw.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (samurang87) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: oops!

**Pull request merged**